### PR TITLE
More init clean up

### PR DESCRIPTION
**Pull request by AJenbo**
... again half of the remaining, at this rate it will take infinite time to finish.

### CODE REVIEW

File: Source/lighting.cpp [hunk 1]
```diff
@@ -493,10 +493,6 @@ char GetLight(int x, int y)
 
 void DoLighting(Point position, int nRadius, int Lnum)
 {
-	int x, y, v, mult, radius_block;
-	int min_x, max_x, min_y, max_y;
-	int dist_x, dist_y, temp_x, temp_y;
```
Comment (julealgon) [inline Source/lighting.cpp:498]: ...lol at the amount of variables. This code man...

File: Source/msg.cpp [hunk 2]
```diff
@@ -804,15 +797,15 @@ void DeltaLoadLevel()
...
-						for (j = -k; j <= k && !done; j++) {
-							yy = y + j;
-							for (l = -k; l <= k && !done; l++) {
-								xx = x + l;
+					bool done = false;
+					for (int k = 1; k < 50 && !done; k++) {
+						for (int j = -k; j <= k && !done; j++) {
+							int yy = y + j;
+							for (int l = -k; l <= k && !done; l++) {
+								int xx = x + l;
```
Comment (julealgon) [inline Source/msg.cpp:808]: My god...

**Review by julealgon — APPROVED** [review]

File: Source/msg.cpp (hunk 2 above)
Comment (qndel) [reply Source/msg.cpp:808]: he's not even yours! 😃

Comment (julealgon) [reply Source/msg.cpp:808]: This might be the most indented code I've ever seem in a real code actually... now that I think about it.

Comment (AJenbo) [reply Source/msg.cpp:808]: Still two less then what I have to deal with professionally.

**Pull request merged**